src/chrome/content/rules/topachat.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Connection refused:
+		topachat.com
```

Comment:
Could you add a redirect from `^http://topachat\.com/` to `https://www.topachat.com/`?